Uno.Gallery/Uno.Gallery.Shared/Extensions/ControlExtensions.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using Windows.UI.Xaml;
+
+namespace Uno.Gallery
+{
+	/// <summary>
+	/// Helper class for Controls Extensions.
+	/// </summary>
+	public static class ControlExtensions
+	{
+		public static readonly DependencyProperty ResourcePathProperty =
+		DependencyProperty.RegisterAttached(
+			"ResourcePath",
+			typeof(string),
+			typeof(ControlExtensions),
+			new PropertyMetadata(null, OnResourcePathChanged));
+
+		public static void SetResourcePath(FrameworkElement element, string value)
+		{
+			element.SetValue(ResourcePathProperty, value);
+		}
+
+		public static string GetResourcePath(FrameworkElement element)
+		{
+			return (string)element.GetValue(ResourcePathProperty);
+		}
+
+		private static void OnResourcePathChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+		{
+			var control = d as FrameworkElement;
+			if (control != null)
+			{
+				var resourcePath = (string)e.NewValue;
+				AddResourcesFromPath(control, resourcePath);
+			}
```

Comment:
you should allow it to reset, when assigning null
```suggestion
			if (d is not FrameworkElement fe)
			{
				fe.Resources = e.NewValue is string path
					? new ResourceDictionary() { Source = new Uri(resourcePath, UriKind.RelativeOrAbsolute) }
					: default;
			}
```

Reply:
I didnt exactly take your code because I think you made a mistake and wrote "is not Fr..." instead of "is Frame..." what I have right now is good, right?